llvm/lib/Analysis/StaticDataProfileInfo.cpp

```diff
@@ -46,6 +50,12 @@ bool IsAnnotationOK(const GlobalVariable &GV) {
 } // namespace memprof
 } // namespace llvm
 
+#ifndef NDEBUG
+static StringRef debugPrintSectionPrefix(StringRef Prefix) {
```

Comment:
This is only used within one function, so maybe move it closer to where it's used and make it a function object?